tools/taskcluster/decision_task.py

```diff
@@ -0,0 +1,70 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import json
+import os
+
+import taskcluster
+
+from tasks import schedule_task_graph, TaskBuilder
+
+
+def pr(builder, branch):
+    return (taskcluster.slugId(), builder.craft_pr_task(branch)),
+
+
+def master(builder):
+    return (taskcluster.slugId(), builder.craft_master_task()),
+
+
+def release(builder, tag):
+    return (taskcluster.slugId(), builder.craft_release_task(tag)),
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(dest='command')
+
+    pr_parser = subparsers.add_parser('pull-request')
+    pr_parser.add_argument('author')
+    pr_parser.add_argument('branch')
```

Comment:
What's the driver for changing the CLI API? As far I understand, Chain of Trust expects the env var `MOBILE_HEAD_REF` to rebuild the context. This will be an issue whenever we turn push-apk on.

Not a blocker.

Reply:
Ooh, good point about Chain of Trust. My rationale for this change is that it becomes more obvious which values are required, and in which situations - for example, I don't think the `MOBILE_HEAD_REF` is required (by the decision task) except for PRs.

I'm using dep signing and push workers right now without Chain of Trust issues, but that's probably because they're dep. If I run into issues when I use production scriptworkers, I'll add the environment variables accordingly.